assets/src/edit-story/components/keyboard/index.js

```diff
@@ -0,0 +1,403 @@
+/*
+ * Copyright 2020 Google LLC
```

Comment:
Isn't all this a duplicate from `design-system/components/keyboard/index.js`? Shouldn't it be used directly from there instead? Or what's the idea of having it in both places? Thoughts?

Reply:
Looks like a leftover file from fixing merge conflicts after my refactoring.

Reply:
Ah, I thought I fixed all the merge problems - will delete this too.

Reply:
Removed in eb983d560.